tests/preprocessing/test_gene_scoring.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import scanpy as sc
+
+import scib
+
+
+def test_mouse(adata_paul15):
+    scib.pp.score_cell_cycle(
```

Comment:
Could you add some asserts to the test function itself? E.g. that columns `["S_score", "G2M_score", "phase"]` were added to the adata